car.capnp

```diff
@@ -424,6 +424,7 @@ struct CarParams {
   safetyConfigs @62 :List(SafetyConfig);
   alternativeExperience @65 :Int16;      # panda flag for features like no disengage on gas
   maxLateralAccel @68 :Float32;
+  autoResumeSng @69 :Bool;
```

Comment:
```suggestion
  autoResumeSng @69 :Bool;  # describes whether car can resume from a stop automatically
```

Reply:
also add something saying it's only used for docs